pkg/registries/registries.go

```diff
@@ -3,17 +3,48 @@ package registries
 import (
 	"fmt"
 	"sort"
+	"strings"
 
 	config "github.com/openshift/api/config/v1"
 )
 
 // mirror represents a mirrored image repo entry in a registry configuration file
 type mirror struct {
+	// host is the mirror image target location. Can include the registry hostname/IP address, port, and namespace path
```

Comment:
target refers to the fact that containerd will target this location rather than a source location to pull the image. this is info beyond the scope of the struct though so I'll remove this wording